isis/src/base/apps/findimageoverlaps/findimageoverlaps.h

```diff
@@ -0,0 +1,14 @@
+#ifndef findimageoverlaps_h
+#define findimageoverlaps_h
+
+#include "UserInterface.h"
+#include "Pvl.h"
+
+using namespace std;
+using namespace Isis;
```

Comment:
Probably makes sense to take this out, given the `namespace Isis` below.

Reply:
Both of these usings need to be removed. Don't put using statements in header files as they pollute everything that includes them.